Tools/Replay/Replay.cpp

```diff
@@ -538,6 +538,7 @@ bool Replay::find_log_info(struct log_information &info)
                 if (smallest_delta == 0 || delta < smallest_delta) {
                     smallest_delta = delta;
                 }
+                prev = reader.last_clock_timestamp;
```

Comment:
dunno if this is actually right. Without more info on the commit message I'd need to dig into this part of the codebase.

However if it is, then then assignment should be move after the if since it's done in both branches.